### DISCUSSION TITLE
When the table is refetched with tanstack query, it loses paging state

### DISCUSSION THREAD

**Discussion by jesus-osornio** [KevinVandy/material-react-table#852] [Q&A]
When the data of table is refetching with tanstack query the table lose the state of pagination, makes bad user experience resetting state of table.  Exists some alternative without i controlled the  state and it makes automatically ?

**Comment by rdmzcn** [comment]
Did you find a solution?

**Comment by KevinVandy** [comment]
You're probably unmounting the table components. None of the examples have this problem.
